poller-lambdas/src/pollers/reuters/auth.ts

```diff
@@ -0,0 +1,39 @@
+import { z } from 'zod';
+
+const AuthSchema = z.object({
+	access_token: z.string(),
+	expires_in: z.number(),
+	token_type: z.string(),
+});
+
+const scopes =
+	'https://api.thomsonreuters.com/auth/reutersconnect.contentapi.read https://api.thomsonreuters.com/auth/reutersconnect.contentapi.write';
+const authUrl = 'https://auth.thomsonreuters.com/oauth/token';
+const grantType = 'client_credentials';
+const audience = '7a14b6a2-73b8-4ab2-a610-80fb9f40f769';
```

Comment:
just double checking that this value is fine to be made public? iirc it's a value that's shared by all customers, is that right?

Reply:
yep that's right -- it's from the Reuters API documentation 👍